pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -267,7 +280,13 @@ func describeAdditionalBuildDetail(build *graph.BuildConfigNode, includeSuccess
 	return out
 }
 
-func describeBuildStatus(build *buildapi.Build, t *util.Time, parentName string) string {
+func describeBuildStatus(build *buildapi.Build, t *util.Time, parentName string, pushTarget *graph.ImageStreamNode) string {
+	imageStreamFailure := ""
+	// if we're using an image stream and that image stream is the internal registry and that registry doesn't exist
+	if (pushTarget != nil) && (build.Parameters.Output.To != nil) && (len(pushTarget.ImageStream.Status.DockerImageRepository) == 0) {
+		imageStreamFailure = fmt.Sprintf(" (is/%s missing docker registry: oadm registry)", build.Parameters.Output.To.Name)
```

Comment:
Too much info in too little space.  I think this would be more effective if you just had "can't push to image" and then at the end of the output if any of the image streams have this condition, recorded "Warning: Some of your builds are pointing to image streams, but the administrator has not configured the integrated Docker registry".

That also implies that you want to be able to look at the returned groups as a unit and quick assess that condition (rather than having to return values from describe all the way up).